opentelemetry-api/src/opentelemetry/propagation/__init__.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import typing
+
+import opentelemetry.context.propagation.httptextformat as httptextformat
+import opentelemetry.trace as trace
+from opentelemetry.context import Context
+from opentelemetry.context.propagation import (
+    ContextT,
+    DefaultHTTPExtractor,
+    DefaultHTTPInjector,
+    Getter,
+    Setter,
+)
+
+
+def extract(
+    carrier: ContextT,
+    context: typing.Optional[Context] = None,
+    extractors: typing.Optional[
+        typing.List[httptextformat.HTTPExtractor]
```

Comment:
Nit: this should probably be `Sequence`, like many of our other `List`s in type annotations.